src/SprykerEco/Zed/UnzerApi/Business/Api/Request/UnzerApiRequestConstants.php

```diff
@@ -0,0 +1,441 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Zed\UnzerApi\Business\Api\Request;
+
+interface UnzerApiRequestConstants
```

Comment:
Please get rid of non-public API constants files:
1. Please use {Transfer}::PROPERTY where possible. And protected constants directly in the mapper were not.

Reply:
Can we skip it for this release?

Reply:
Discussed, we can skip it because of timelines.